### PR TITLE
Better version control (like in pithos-app)

### PR DESCRIPTION
**Pull request by alexey-medvedchikov**
Gravity 0.0.30 required

### CODE REVIEW

**Comment by sofuture** [comment]
Do we really want to always build the tag 'latest' instead of a specific version? I could see either way, but curious for your reasons or thoughts @alexey-medvedchikov 

:shipit: in general though

**Comment by sofuture** [comment]
I think I am pro `:latest` now, in conjunction with `--set-image` (doing this in pithos-app right now).

File: Makefile [hunk 1]
```diff
@@ -6,3 +12,11 @@ package:
 deploy:
 	$(MAKE) -C images deploy
 
+.PHONY: import
+import: package
+	-gravity app delete --ops-url=$(OPS_URL) $(REPOSITORY)/$(NAME):$(VERSION) \
+		--force --insecure
+	gravity app import --vendor --glob=**/*.yaml --registry-url=apiserver:5000 \
+		--ops-url=$(OPS_URL) --repository=$(REPOSITORY) --name=$(NAME) \
+		--version=$(VERSION) --insecure .
```
Comment (klizhentas) [inline Makefile:22]: I would also add `--set-image=log-collector:$(VERSION)` and `--set-image=log-forward:$(VERSION)` to set proper image names during build.

This also means that during container builds you'd need to tag resulting image both as `log-forward:$(VERSION)` - to vendor it and `log-forward:latest` to use in development mode directly.

**Comment by alexey-medvedchikov** [comment]
@sofuture there's more: it's not working that way. Fixed by passing VERSION from root Makefile

**Comment by sofuture** [comment]
shipit

**Pull request merged**